src/networks.json

```diff
@@ -2111,21 +2111,21 @@
     "start": 43317,
     "logo": "ipfs://QmXvum7SNVaAqAc2jNzR1NpNZN1GGvNaKWydg8a1GEDQ7y"
   },
-  "9000": {
-    "key": "9000",
+  "6102": {
+    "key": "6102",
     "name": "Cascadia Testnet",
     "shortName": "Cascadia",
-    "chainId": 9000,
+    "chainId": 6102,
     "network": "testnet",
-    "multicall": "0xBd186aca63bD4f5f4eC9E98028461A6A94ac4a12",
+    "multicall": "0x728989819bAD588F193563008E0a03E8cD6a3e4a",
     "rpc": [
-      "https://devnet.cascadia.foundation"
+      "https://testnet.cascadia.foundation"
     ],
     "explorer": {
       "url": "https://explorer.cascadia.foundation"
     },
-    "start": 694000,
-    "logo": "ipfs://QmRgv8ckPdzwHKehJphKDcE3rfuNQaJzjSWq2nuVTTMepZ"
+    "start": 0,
```

Comment:
```suggestion
    "start": 370457,
```

Reply:
You want us to set the start to 370457 ?

Reply:
Yep

Reply:
Does it affect something if we don't start at 0 ?

Reply:
Yes sir, this way we know when multicall is deployed

Reply:
Ok Sir,
I have completed everything and Commited I guess. Right ? 
Anything I need to do more ?